libpod/runtime_pod.go

```diff
@@ -93,3 +96,41 @@ func (r *Runtime) Pods(filters ...PodFilter) ([]*Pod, error) {
 
 	return podsFiltered, nil
 }
+
+// GetAllPods retrieves all pods
+func (r *Runtime) GetAllPods() ([]*Pod, error) {
+	r.lock.RLock()
+	defer r.lock.RUnlock()
+
+	if !r.valid {
+		return nil, ErrRuntimeStopped
+	}
+
+	pods, err := r.state.AllPods()
+	if err != nil {
+		return nil, err
+	}
+
+	return pods, nil
+}
+
+// GetLatestPod returns a pod object of the latest created pod.
+func (r *Runtime) GetLatestPod() (*Pod, error) {
+	lastCreatedIndex := -1
+	var lastCreatedTime time.Time
+	pods, err := r.GetAllPods()
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to find latest pod")
```

Comment:
i wonder if improving the error message would be more helpful should we ever need to debug.  here i think the issue is you couldn't get the pods ... take the comment as you wish.